vllm/transformers_utils/tokenizer.py

```diff
@@ -167,7 +167,7 @@ def get_lora_tokenizer(lora_request: LoRARequest, *args,
         return None
     try:
         tokenizer = get_tokenizer(lora_request.lora_path, *args, **kwargs)
-    except OSError as e:
+    except (OSError, ValueError) as e:
```

Comment:
`ValueError` rather than `OSError` is now thrown when there's no config.json present